xbmc/GUIInfoManager.cpp

```diff
@@ -7223,88 +7190,88 @@ bool CGUIInfoManager::GetBool(int condition1, int contextWindow, const CGUIListI
     CGUIWindowSlideShow *slideShow = g_windowManager.GetWindow<CGUIWindowSlideShow>(WINDOW_SLIDESHOW);
     bReturn = (slideShow && slideShow->GetCurrentSlide() && slideShow->GetCurrentSlide()->IsVideo());
   }
-  else if (g_application.m_pPlayer->IsPlaying())
+  else if (g_application.GetAppPlayer().IsPlaying())
```

Comment:
why did you keep this?

Reply:
I'm not sure if that is fine to remove all `IsPlaying` check in `CGUIInfoManager `

Reply:
dropped all `IsPlaying` checks from GUIInfoManager .. runtime tested and it looks good.